_includes/com-install/com-install-startup-fb.md

```diff
@@ -1,4 +1,4 @@
-* Open a terminal and CD to the `/featurebase/opt` directory.
+* Open a terminal and CD to `/featurebase/opt`.
```

Comment:
Not sure I understand this change. We should avoid confusion that /featurebase is at the root of your computer. Another alt would be */featurebase/opt

Reply:
Fixed with * in front through help